workflow/artifacts/artifactory/artifactory_test.go

```diff
@@ -0,0 +1,57 @@
+package artifactory_test
+
+import (
+	art "../artifactory"
+	wfv1 "github.com/argoproj/argo/api/workflow/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+)
+
+const (
+	LoadFileName string = "argo_artifactory_test_load.txt"
+	SaveFileName string = "argo_artifactory_test_save.txt"
+	RepoName     string = "generic-local"
+	URL          string = "http://localhost:8081/artifactory/" + RepoName + "/" + LoadFileName
+	Username     string = "admin"
+	Password     string = "password"
+)
+
+func TestSaveAndLoad(t *testing.T) {
+
+	t.Skip("This test is skipped since it depends on external service")
```

Comment:
Can we make this test use some external flag that the user passes from the command line? If that flag is absent, the test can be skipped. If it is present, then the test assumes that artifactory is installed and run the full tests.

This way, no code changes need to be made for the test to run. What do you think?

Reply:
This is a good idea but the end result is still that i need to skip this test. Will add the flag next time i have a need to uncomment the test.

Reply:
Can you file a bug for this please so that it doesn't get forgotten?

Reply:
sure. filed https://github.com/argoproj/argo/issues/650